prow/test/integration/test/horologium_test.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	uuid "github.com/satori/go.uuid"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestLaunchProwJob(t *testing.T) {
+	const existJobName = "horologium-schedule-test-job"
+	t.Parallel()
+
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "running-pod",
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			clusterContext := getClusterContext()
+			t.Logf("Creating client for cluster: %s", clusterContext)
+			kubeClient, err := NewClients("", clusterContext)
+			if err != nil {
+				t.Fatalf("Failed creating clients for cluster %q: %v", clusterContext, err)
+			}
+			ctx := context.Background()
+
+			getNextRunOrFail := func(t *testing.T, jobName string, lastRun *v1.Time) *prowjobv1.ProwJob {
+				var res *prowjobv1.ProwJob
+				if err := wait.Poll(time.Second, 70*time.Second, func() (bool, error) {
+					pjs := &prowjobv1.ProwJobList{}
+					err = kubeClient.List(ctx, pjs, ctrlruntimeclient.InNamespace(defaultNamespace))
```

Comment:
Instead of doing a full list, can we just list with a label selector of the job name? That would allow us to not block so much, horologium will create the job as soon as its up and if the job exist, we know the test passed.

I'd also be in favor of moving the prow-cm-tests into a dedicated PR and scoping this to "make sure horologium creates a prowjob"

Reply:
Done, both